Networking/NetworkingTests/Remote/WooShippingRemoteTests.swift

```diff
@@ -383,6 +383,36 @@ final class WooShippingRemoteTests: XCTestCase {
         // Then
         XCTAssertNotNil(result.failure)
     }
+
+    func test_loadOriginAddresses_parses_success_response() throws {
```

Comment:
Could we also add a test here for a failure response?

Reply:
Done https://github.com/woocommerce/woocommerce-ios/pull/14689/commits/261ad63ae6ea8d699db9878b04aae15bc8da47c4